src/components/PokemonFiche.js

```diff
@@ -0,0 +1,46 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+
+
+function PokemonFiche({name}) {
+
+    const [pokemonData, setPokemonData] = useState([]);
+
+    useEffect(() => {
+        async function getOnePokemon() {
+            try {
+                const response = await axios.get(`https://pokeapi.co/api/v2/pokemon/${name}`);
+                console.log(response);
+                setPokemonData(response);
+            } catch (e) {
+                console.error(e)
+            }
+        }
+
+        getOnePokemon();
+    }, [])
+    return (
+        <>
+            {Object.keys(pokemonData).length > 0 &&
```

Comment:
Dan kan deze check ook simpeler
```suggestion
            {pokemonData &&
```